Unix/tests/http/test_http.cpp

```diff
@@ -730,12 +732,14 @@ static void _ConnectToServerExpectConnectionDrop(const string& data, MI_Uint32 s
         // read response
         char r_buf[1024];
         size_t read = 0;
+        int err;
 
         do
         {
             r = Sock_Read(sock, r_buf, sizeof(r_buf), &read);
+            err = Sock_GetLastError();
         }
-        while (r != MI_RESULT_OK && Sock_GetLastError() == 11 /*EAGAIN*/);
+        while (r != MI_RESULT_OK && (err == 11 || err == 35) /*EAGAIN*/);
```

Comment:
That's actually much better (use the #define value). Allows for self-documenting code ...